index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/6d267a36ba.js" crossorigin="anonymous"></script>
+    <title>Portfolio Template</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="main">
+        <div>
+            <nav>
+                <ul class="menu">
+                    <li>Portfolio</li>
+                    <li>About</li>
+                    <li>Contact</li>
+                </ul>
+            </nav>
+            <div class="h-menu">
+                <h1>Welcome</h1>
+                <div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+                </div>
+            </div>
+            <div class="body">
+                <br>
+                <br>
+                <br>
+                <h1>Hey there. I’m Yegon</h1>
+                <h2>I’m a software developer</h2>
+                <br>
+                <br>
```

Comment:
- [ ] Just as we agreed, kindly use `css` properties like padding or margin to avoid repetition. This makes your code very clean and professional.